src/itemLogin/factory.ts

```diff
@@ -7,6 +11,7 @@ export function ItemLoginSchemaFactory(
   return {
     id: faker.string.uuid(),
     type: ItemLoginSchemaType.Username,
+    state: ItemLoginSchemaState.Active,
```

Comment:
I think I prefer status.
```suggestion
    status: ItemLoginSchemaStatus.Active,
```